.github/workflows/proof_ci.yaml

```diff
@@ -114,6 +130,36 @@ jobs:
           sudo apt-get update
           sudo apt-get install --no-install-recommends --yes ./litani.deb
           rm ./litani.deb
+      - name: Install Z3 ${{ env.Z3_VERSION }}
+        if: ${{ env.Z3_VERSION != 'latest' }}
```

Comment:
Does this actually make sense given the above checks?

Reply:
I decided to leave that test in so we can move to supporting a "latest" Z3 in the future.